roxctl/netpol/connectivity/diff/cta.go

```diff
@@ -24,6 +25,9 @@ type diffAnalyzer interface {
 func (cmd *diffNetpolCommand) analyzeConnectivityDiff(analyzer diffAnalyzer) error {
 	connsDiff, err := analyzer.ConnDiffFromDirPaths(cmd.inputFolderPath1, cmd.inputFolderPath2)
 	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			return errox.NotFound.Newf(err.Error())
+		}
```

Comment:
Thank you for wapping the error. Unfortunately the message is still hard to understand:

```
ERROR:  error accessing directory: lstat /nonexisting: no such file or directory
```

Could you guess which dir is missing? dir1 or dir2?

Spoiler alert: this issue is unfixable with the current state of NP-Guard code (i.e. it is impossible to tell which of those two folders is missing). I
n order to fix it, NP-Guard would need to accept string inputs instead of relying on folders. Sorry if I repeat it too frequently, but I really think the inputs in form of folders should be abandoned.

Reply:
@vikin91 , given the schedule and the required effort to improve the error message, can we delay handling this issue to the next release? 
This error is definitely not the ideal UX, but it is also not a huge effort for the user to find out whether the error relates to dir1 or dir2.